Mlem/Views/Shared/Links/User Profile Label.swift

```diff
@@ -10,17 +10,17 @@ import CachedAsyncImage
 
 struct UserProfileLabel: View {
     @AppStorage("shouldShowUserAvatars") var shouldShowUserAvatars: Bool = true
-
-    @State var account: SavedAccount
-    @State var user: APIPerson
-    @State var showServerInstance: Bool
-
+    
+    var account: SavedAccount
+    var user: APIPerson
+    var showServerInstance: Bool
+    
     // Extra context about where the link is being displayed
     // to pick the correct flair
     @State var postContext: APIPostView?
```

Comment:
We can probably remove these as states too right?  I don't envision this state changing while this view is up.

Reply:
Probably--leaving as-is for now because it's not breaking anything but there's a @State audit planned for 1.1